### PR TITLE
Test/sign up

### PR DESCRIPTION
**Pull request by cvillamer13**
sir pacheck ako ginawa ko dito same lang 1 and 2 pa din sa role nila

### CODE REVIEW

File: src/services/user.service.ts [hunk 1]
```diff
@@ -21,12 +21,11 @@ export async function signUp(data: ISignUp) {
   logger.info('User signing up ...')
 
   try {
-    const result: UserCredential = await createUserWithEmailAndPassword(
+    const result: any = await createUserWithEmailAndPassword(
```
Comment (aldenvallestero) [inline src/services/user.service.ts:24]: Why did you removed the **UserCredential**?

File: src/tests/fixtures/user.fixture.ts [hunk 2]
```diff
@@ -2,3 +2,11 @@ export const signInFixture = {
   email: 'aldenvallestero@icloud.com',
   password: 'denonjuancareerS10!',
 }
+
+
+export const signUpFixture = {
+  email: 'cvillamer31@gmail.com',
```
Comment (aldenvallestero) [inline src/tests/fixtures/user.fixture.ts:8]: - [ ] use fakerjs library to generate an email.
- [ ] use uuid version 4 library to generate a password.

File: src/services/user.service.ts (hunk 1 above)
Comment (cvillamer13) [reply src/services/user.service.ts:24]: Hindi naman ginagamit kaya ginawa ko any  makuha yung result

Comment (aldenvallestero) [reply src/services/user.service.ts:24]: okay

File: src/tests/fixtures/user.fixture.ts [hunk 3]
```diff
@@ -2,3 +2,11 @@ export const signInFixture = {
...
+  password: '123456',
```
Comment (aldenvallestero) [inline src/tests/fixtures/user.fixture.ts]: - [ ] use **uuid version 4** library to generate a password.

File: src/tests/fixtures/user.fixture.ts (hunk 2 above)
Comment (aldenvallestero) [inline src/tests/fixtures/user.fixture.ts]: - [ ] use **fakerjs** library to generate an email.

**Review by aldenvallestero — APPROVED** [review]

**Pull request merged**